components/brave_wallet/browser/internal/hd_key.cc

```diff
@@ -42,6 +38,13 @@
 using crypto::Encryptor;
 using crypto::SymmetricKey;
 
+namespace base {
```

Comment:
This is definitely not something we should be doing. You're effectively creating a layering violation with a kind of backdoor chromium src override for base.

Reply:
I basically wanted instances of this
```
template <size_t N>
struct SecureByteArray : public std::array<uint8_t, N> {
  ~SecureByteArray() { crypto::internal::SecureZeroBuffer(base::span(*this)); }
};

```
to be implicitly convertible to fixed size spans. 

Will think how to achieve this in a different but succinct way.